.github/workflows/ci.yml

```diff
@@ -46,15 +46,26 @@ jobs:
             blog/**/*.{md,mdx}
 
       - name: Publish to Hashnode and Dev.to 📝
-        if: github.ref == 'refs/heads/develop' && steps.changed-markdown-files.outputs.any_changed == 'true'
         env:
           HASHNODE_PAT: ${{ secrets.HASHNODE_PAT }}
           HASHNODE_PUBLICATION_ID: ${{ secrets.HASHNODE_PUBLICATION_ID }}
-          CHANGED_FILES: ${{ steps.changed-markdown-files.outputs.all_changed_files }}
+          ADDED_FILES: ${{ steps.changed-markdown-files.outputs.added_files }}
```

Comment:
I don't think we need this. The JS code can automatically just pull all the files and identify what has changed.